webapp/src/Controller/API/UserController.php

```diff
@@ -291,12 +291,12 @@ public function singleAction(Request $request, string $id): Response
      * Add a new user.
      */
     #[IsGranted('ROLE_API_WRITER')]
-    #[Rest\Post]
+    #[Rest\Put("")]
```

Comment:
I think it's fine to also allow PUT here, but dropping POST doesn't make sense to me.